server/helpers/utils/utils.go

```diff
@@ -353,3 +356,48 @@ func MarshalAndUnmarshal[k any, v any](val k) (unmarshalledvalue v, err error) {
 	}
 	return
 }
+
```

Comment:
Where are the new error codes with full descriptions?

Reply:
@leecalcote we don't need new error codes to be defined, most of them are already defined and used in connection handlers.